packages/@cdktf/provider-generator/lib/get/__tests__/generator/__snapshots__/module-generator.test.ts.snap

```diff
@@ -8354,6 +8490,11 @@ export interface ModuleConfig extends TerraformModuleUserConfig {
    */
   readonly fetchCallerIdentity?: boolean;
 }
+/**
+ * Defines an Module based on a Terraform module
+ *
+ * Source at ./module
```

Comment:
Isn't this file in e.g. `./.gen/modules/Module.ts` or the like and the module would be in `./module`, so the path is incorrect, isn't it?

Reply:
Unfortunately, I don't have much information about where the module code is really coming from. That said, I'm rendering a docstring for the `Module.ts` file here, so the code should actually be located next to the generated file as it is also passed like that to the modules `source` attribute a few lines further down in the module generator.